app/models/katello/module_stream.rb

```diff
@@ -0,0 +1,44 @@
+module Katello
+  class ModuleStream < ApplicationRecord
+    include Concerns::PulpDatabaseUnit
+    has_many :repository_module_stream, class_name: "Katello::RepositoryModuleStream",
+      dependent: :destroy, inverse_of: :module_stream
+    has_many :repositories, through: :repository_module_stream, class_name: "Katello::Repository"
```

Comment:
What benefit does `through: :repository_module_stream` provide? I'm fairly sure you could 'go direct' and drop `repository_module_stream`.

Reply:
Don't worry - though this said `belongs_to`. me--